lib/mongoid/contextual/none.rb

```diff
@@ -88,7 +88,7 @@ def exists?; false; end
       # @example Get the values for null context.
       #   context.pluck(:name)
       #
-      # @param [ String | Symbol ] *_fields Field or fields to pluck.
+      # @param [ String | Symbol ] *_fields Field(s) to pluck.
```

Comment:
Looks like we're inconsistent about how we manage * fields

Reply:
See above comment. Note in diff the `*` isn't changing here, I just changed the text (for consistency)